content/docs/guides/continuous-delivery/spinnaker.md

```diff
@@ -36,42 +42,61 @@ To sign-up for a new Pulumi account, head over to the [Pulumi Console](https://a
 The sample pipeline (shown later) below acts on a hypothetical stack: `dev`.
 
 * Depending on your use-case, you may want to create the Pulumi Stack ahead of time.
-* To create a new stack, you have a couple of options, you may create it in the [Pulumi Console](https://app.pulumi.com/site/new-project) directly.
-* Alternatively, you can also run `pulumi new [template]` to create a [template project]({{< relref "/docs/reference/cli/pulumi_new" >}}).
+* You can create a new stack in the [Pulumi Console](https://app.pulumi.com/site/new-project).
+* Alternatively, you can also run `pulumi new [template]` to create a [template project]({{< relref "/docs/reference/cli/pulumi_new" >}})
+on your machine and push it up to your VCS repo.
   
-The source code for the stack is in a repository in GitHub cloned from this [example](https://github.com/pulumi/examples/tree/master/kubernetes-ts-nginx) and uses `TypeScript` as the language.
-The example infrastructure code creates a Kubernetes `Deployment` using the `nginx` container image.
-If you modify the sample app or wish to create the `Deployment` resource in a different Kubernetes cluster, follow the setup guide for the [Kubernetes provider](https://www.pulumi.com/docs/intro/cloud-providers/kubernetes/setup/)
+The source code used as an [example](https://github.com/pulumi/examples/tree/master/kubernetes-ts-nginx) in this guide
+uses `TypeScript`. The example infrastructure code creates a Kubernetes `Deployment` using the `nginx` container image.
+If you modify the sample app or wish to create the `Deployment` resource in a different Kubernetes cluster, follow the
+setup guide for the [Kubernetes provider](https://www.pulumi.com/docs/intro/cloud-providers/kubernetes/setup/)
 to ensure that Pulumi can access that Kubernetes cluster.
 
-> While the example shows you how to deploy a Kubernetes resource, there are many other [cloud providers](https://www.pulumi.com/docs/intro/cloud-providers/) that Pulumi supports.
+> While the example shows you how to deploy a Kubernetes resource, there are many other [cloud providers](https://www.pulumi.com/docs/intro/cloud-providers/)
+> that Pulumi supports.
 
 **Note**: The names used above are purely for demonstration purposes only.
 You may choose a naming convention that best suits your organization.
```

Comment:
This probably doesn't need to be a note. Also, which "names" are we referring to here? The stack name? If so, then we can append this sentence to the sentence that talks about the stack name.

Reply:
I'll move it up to the sentence about stack name and rephrase it a bit.